Exiled.Events/EventArgs/Scp106/PlayerTryEnterStalkEventArgs.cs

```diff
@@ -0,0 +1,88 @@
+// -----------------------------------------------------------------------
+// <copyright file="PlayerTryEnterStalkEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs.Scp106
+{
+    using API.Features;
+    using Interfaces;
+    using Mirror;
+    using PlayerRoles.PlayableScps.Scp106;
+    using PlayerRoles.PlayableScps.Subroutines;
+    using UnityEngine;
+
+    /// <summary>
+    ///     Contains all information before SCP-106 teleports using a portal.
+    /// </summary>
+    public class PlayerTryEnterStalkEventArgs : IPlayerEvent, IDeniableEvent
```

Comment:
Rename to StalkingEventArgs

Reply:
@Thundermaker300 The problem is that, it's not that they're actual stalking, it's the player (client) requesting to change their stalk status. The client will wait on an answer from the server; therefore, it's a client interaction. If that still means changing to StalkingEventArgs, let me know; however, I feel as if there should be an indication that yeah, this is a client request that you can reject

Reply:
I see. Maybe something like `RequestingStalkEventArgs`? I feel as though having `Player` in the name is redundant since it's an `IPlayerEvent`.

Reply:
Done